node-fs-writefile/cp.js

```diff
@@ -0,0 +1,13 @@
+import { writeFile, readFile } from 'node:fs/promises';
+
+async function cp() {
+  try {
+    const contents = await readFile(process.argv[2], 'utf8');
+    const copyFile = String(process.argv[3]);
```

Comment:
argv elements are always strings, so converting it to a string with `String()` is not necessary.

Reply:
removed unnecessary `String()`